x/gov/client/cli/parse.go

```diff
@@ -19,17 +19,41 @@ type legacyProposal struct {
 	Deposit     string
 }
 
+func (p legacyProposal) validate() error {
+	if p.Type == "" {
+		return fmt.Errorf("proposal type is required")
+	}
+
+	if p.Title == "" {
+		return fmt.Errorf("proposal title is required")
+	}
+
+	if p.Description == "" {
+		return fmt.Errorf("proposal description is required")
+	}
+	return nil
+}
+
 func parseSubmitLegacyProposalFlags(fs *pflag.FlagSet) (*legacyProposal, error) {
 	proposal := &legacyProposal{}
 	proposalFile, _ := fs.GetString(FlagProposal)
 
 	if proposalFile == "" {
 		proposalType, _ := fs.GetString(FlagProposalType)
+		title, _ := fs.GetString(FlagTitle)
+		description, _ := fs.GetString(FlagDescription)
+		if proposalType == "" && title == "" && description == "" {
```

Comment:
do we still need this? `validate` is already doing this right?

Reply:
so this error would say one of three have an issue but the other error would say specifically what the error is right?